grommet-leaflet/src/layers/Map/Map.jsx

```diff
@@ -22,6 +23,7 @@ const Map = forwardRef(
       scrollWheelZoom = true,
       tileLayer,
       theme,
+      maxZoom = 18,
```

Comment:
Can you add a comment for what "18" equates to? Or a link to the legen

Reply:
added comments and link to docs